bot/cogs/grades_cog.py

```diff
@@ -0,0 +1,259 @@
+import json
+import os
+
+import logging
+import random
+import time
+
+import discord
+import discord.ext.commands as commands
+
+from bot.consts import Colors
+
+log = logging.getLogger(__name__)
+
+
+
+class gradesCog(commands.Cog):
+    special_converted_files = ['2018Fall.csv','2018Spring.csv'] #these files have special formatting for names (ie ' "Smith, John William" ' is normal and in these it's simply 'Smith John William' without commas or quotes )
+    #TBH, this is the easier way to parse but whatevs
+    converted_files = ['2013Fall.csv', '2014Spring.csv', '2014Fall.csv', '2015Spring.csv', '2015Fall.csv', '2016Spring.csv', '2016Fall.csv', '2017Spring.csv', '2017Fall.csv', '2019Spring.csv', '2019Fall.csv', '2020Spring.csv']
+
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.fileLines = []
+        self.master_list = {}
+        self.master_prof_list = {}
+
+
+
+    def process_Search(self, orig_query: str) -> str: #Primary search function and processing for a query. A query is generally defined by 
+                                    #"Course-Number" of which it pulls the data from the DB.
+        print("PROCESSING")
```

Comment:
Prints are kinda useless in clembot, this can either be removed or turned into a more descriptive log.Info statement